homeassistant/components/vicare/sensor.py

```diff
@@ -661,7 +661,7 @@ class ViCareSensorEntityDescription(SensorEntityDescription, ViCareRequiredKeysM
     ),
     ViCareSensorEntityDescription(
         key="pcc_transfer_power_exchange",
```

Comment:
I did not sync the key to avoid any delay. Feel free to adjust if this makes any sense.

Reply:
The side effect would be that the unique ID changes. It is nicer to adjust this and migrate existing entities in the registry of users to the new ID

Reply:
This would mean adding new code which needs coverage targets to fulfill, I would like to avoid this for now.

Reply:
Honestly, I think it is better to add it now. It will be a hurdle to add it later. 🤷

Reply:
Left key and translation key unchanged.